recipes/openslide/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set OPENSLIDE_VERSION = "3.4.1" %}
+
+package:
+  name: openslide
+  version: {{OPENSLIDE_VERSION}}
+
+source:
+  url: https://github.com/openslide/openslide/releases/download/v{{OPENSLIDE_VERSION}}/openslide-{{OPENSLIDE_VERSION}}.tar.gz
+  md5: fd69a74481a57d6dc188c7a276a09ee7
+
+build:
+  number: 0
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+  host:
+    - cairo >=1.2
+    - gdk-pixbuf
+    - glib >=2.48.0
+    - jpeg
+    - openjpeg >=2.1
+    - libpng
+    - libtiff >=4
+    - libxml2
+    - sqlite >=3.6.2
+    - xorg-libxrender
+    - xorg-libxext
+    - xorg-libxau
+    - xorg-libxdmcp
+    - zlib
+  run:
+    - cairo >=1.2
+    - gdk-pixbuf
+    - glib >=2.48.0
+    - jpeg
+    - libpng
+    - libtiff >=4
+    - libxml2
+    - openjpeg >=2.1
+    - sqlite >=3.6.2
+    - zlib
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libopenslide.so  # [linux]
+    - test -f $PREFIX/lib/libopenslide.dylib  # [osx]
```

Comment:
You can do it just in one linux, like:
```
- test -f $PREFIX/lib/libopenslide${SHLIB_EXT}  # [unix]
```

Reply:
Nice!

Reply:
You will need to add some tests for windows, if it is the same kind of test, you can write something like:
```
    - if not exist %LIBRARY_LIB%\libopenslide.lib  exit 1   # [win]
```